modular_skyrat/modules/sec_haul/code/guns/guns.dm

```diff
@@ -25,7 +25,7 @@
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/glock.dmi'
 	icon_state = "glock"
 	w_class = WEIGHT_CLASS_NORMAL
-	mag_type = /obj/item/ammo_box/magazine/multi_sprite/g17
+	mag_type = /obj/item/ammo_box/magazine/multi_sprite/g17 | /obj/item/ammo_box/magazine/multi_sprite/g18
```

Comment:
This isn't valid, guns don't support two types of mag.

Reply:
Already toooold them